test/00_transform/ConvCorr.cu

```diff
@@ -142,6 +162,19 @@ protected:
   }
 
   void TearDown() override { pb.reset(); }
+
+  void CheckExecSupport() {
```

Comment:
This function can be marked constexpr too

Reply:
Fixed